code/modules/language/common.dm

```diff
@@ -0,0 +1,69 @@
+// 'basic' language; spoken by default.
+/datum/language/common
+	name = "Galactic Common"
+	desc = "The common galactic tongue."
+	speech_verb = "says"
+	whisper_verb = "whispers"
+	key = "0"
+	flags = TONGUELESS_SPEECH
+
+//Syllable Lists
+/*
+	This list really long, mainly because I can't make up my mind about which mandarin syllables should be removed,
+	and the english syllables had to be duplicated so that there is roughly a 50-50 weighting.
+
+	Sources:
+	http://www.sttmedia.com/syllablefrequency-english
+	http://www.chinahighlights.com/travelguide/learning-chinese/pinyin-syllables.htm
+*/
+/datum/language/common/syllables = list(

```

Comment:
On bay (and /vg/?) galactic common is english mixed with chinese (which is how scientists think language is going to go anyway (I think they call it penglish?)) so it's just neat fluff/lore.

Personally I think we should keep it.